SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta.cpp

```diff
@@ -378,6 +379,38 @@ void ViewDelta::ProcessOpList(const OpList& Ops)
 	}
 }
 
+void ViewDelta::GenerateComponentChangesFromSetData(const Worker_ComponentSetAuthorityChangeOp& Op, const EntityView& View,
+													const FComponentSetData& ComponentSetData)
+{
+	// Remove all components that are in the component set, then re-add the ones with data in the op.
+	// If one component is both removed and added then this interpreted as component refresh in the view delta.
+	// Otherwise the component will be removed or add as appropriate.
+
+	const TSet<uint32>& Set = ComponentSetData.ComponentSets[Op.component_set_id];
+
+	// If a component on the entity is in the set then generate a remove operation.
+	auto* Entity = View.Find(Op.entity_id);
```

Comment:
Iirc this goes against our code style guidelines for auto

Reply:
very minor, prefer assignment in the if statement personally (for scoping reasons)

Reply:
I think this is fixed in the latest version.